tools/xcodeproj-shims/WorkspaceSettings.xcsettings

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>BuildSystemType</key>
```

Comment:
note, this may not work with future versions of xcode (if the legacy build system is removed)

Reply:
Yes, this is frustrating. FYI Tulsi uses the old build system too. 

Since we know which subdirectory in intermediates is causing problems for us, we could probably just remove it ourselves, even if we are forced to use the new build system.

Reply:
should this be called out in a readme file or an explicit comment some where? This can be unexpected for new onboard person?

Reply:
I'll add a comment!